test/GuardClauses.UnitTests/GuardAgainstOutOfRangeForEnumerableInt.cs

```diff
@@ -22,13 +22,29 @@ public void ThrowsGivenOutOfRangeValue(IEnumerable<int> input, int rangeFrom, in
             Assert.Throws<ArgumentOutOfRangeException>(() => Guard.Against.OutOfRange(input, nameof(input), rangeFrom, rangeTo));
         }
 
+        [Theory]
+        [ClassData(typeof(IncorrectClassData))]
+        public void ThrowsSelfOwnErrorMessageGivenOutOfRangeValue(IEnumerable<int> input, int rangeFrom, int rangeTo)
+        {
+           var exception = Assert.Throws<ArgumentOutOfRangeException>(() => Guard.Against.OutOfRange(input, nameof(input), rangeFrom, rangeTo, "selfOwnErrorMessage"));
```

Comment:
again, indentation --> please use ReSharper and option "Format Document" to avoid such flaws